app/controllers/thredded/messageboard_groups_controller.rb

```diff
@@ -20,6 +20,15 @@ def create
       end
     end
 
+    def show
+      @group = Thredded::MessageboardGroup.where(id: params[:id])
+      @groups = Thredded::MessageboardGroupView.grouped(
+        policy_scope(Thredded::Messageboard.where(group: params[:id])),
+        user: thredded_current_user
+      )
+      render :show
```

Comment:
Remove this line, Rails does `render action_name` by default

Reply:
Corrected.